StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/FetchInitParamsMessageHandler.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 @available(iOS 15, *)
 class FetchInitParamsMessageHandler: ScriptMessageHandlerWithReply<VoidPayload, FetchInitParamsMessageHandler.Reply> {
     struct Reply: Encodable {
-        let locale: String
+        let locale: String?
```

Comment:
If we can't determine the locale, we should omit it from the update to the web view